core/config/options.go

```diff
@@ -71,6 +91,56 @@ func (opts *Options) List(ctx context.Context, key string) ([][]string, error) {
 	return tuples, nil
 }
 
+// ListFunc returns a closure that returns the set of tuples for the
+// provided key.
+//
+// The configuration option for key must be a set of tuples.
+// ListFunc will panic if the provided key is undefined in the schema or
```

Comment:
Nit: I think there's a bit of a convention in godoc to describe a function's behavior in the present tense, so "ListFunc panics if…" rather than "ListFunc will panic if…".

I'm sure we're not consistent about this in our codebase and I'm not even sure how strong of a convention it really is. Dunno why I noticed it just now. I think at least the standard library usually uses present tense e.g. https://godoc.org/net/http#ServeMux.Handle.